jsr107/src/main/java/org/ehcache/sample/BaseJCacheTester.java

```diff
@@ -0,0 +1,75 @@
+package org.ehcache.sample;
+
+import org.ehcache.config.CacheRuntimeConfiguration;
+import org.ehcache.config.ResourceType;
+import org.ehcache.jsr107.Eh107Configuration;
+import org.slf4j.Logger;
+
+import javax.cache.Cache;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+/**
+ * Created by fabien.sanglier on 10/8/16.
+ */
+public abstract class BaseJCacheTester {
+    private static final Logger LOGGER = getLogger(CreateExtendedJCache.class);
+
+    protected <K, V> boolean simpleIterationCacheTest(Cache<K, V> myJCache, int numberOfIteration, int numberOfObjectPerIteration, int sleepTimeMillisBetweenIterations, KeyValueGenerator<K, V> keyValueGenerator) throws InterruptedException {
```

Comment:
I would rename this test method - I thought it was about iterating entries of the cache - which it is not (and that's good 😉 )